spec/NewTwitchApi/Resources/EventSubApiSpec.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace spec\NewTwitchApi\Resources;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Response;
+use PhpSpec\ObjectBehavior;
+use Psr\Http\Message\ResponseInterface;
+
+class EventSubApiSpec extends ObjectBehavior
+{
+    function let(Client $guzzleClient)
+    {
+        $this->beConstructedWith($guzzleClient);
+    }
+
+    function it_should_subscribe_to_channel_update(Client $guzzleClient, Response $response)
+    {
+        $guzzleClient->send(new Request('POST', 'eventsub/subscriptions', ['Authorization' => 'Bearer TEST_TOKEN']))->willReturn($response);
```

Comment:
You'll need to include the body here for your tests, all of the parameters present in the function should be used here in the manual guzzle client.

Reply:
Yes, I'll add it